src/components/organisms/AdminVenueView/AdminVenueView.scss

```diff
@@ -0,0 +1,52 @@
+@import "scss/constants.scss";
+
+.venue-view {
```

Comment:
These are entirely new styles and there are no duplicates. Sidebar had styles that were used instead of these but now that sidebar is deleted and no longer used, this component needs to implement it's own styles.

Reply:
Cool, sounds good :)